### ISSUE TITLE
self-hosting copy and instructions

### ISSUE THREAD

**Issue by Mitzi-Laszlo** [solid/solid.github.io#151]
in ' use Solid' self-hosting needs to be targetted at non developers and should not take a click away in order to be explained. Don't lead to GitHub page (too much developer-oriented). self host is an important part and text is very dev heavy, need an explanation

**Comment by Vinnl** [comment]
We initially decided Jackson would write this part once his Pod Server is ready to be used, but now that that's been postponed to some point in the future, it might indeed be good to add instructions for node-solid-server.
